doc/introduction/interfaces.rst

```diff
@@ -3,17 +3,99 @@
 
 .. _intro_interfaces:
 
-Interfaces and training
-=======================
+Gradients and training
```

Comment:
I would lean towards "Gradients and interfaces" as well here.

Reply:
```suggestion
Gradients and interfaces
```

Although be good to get @mariaschuld's thoughts before switching it!

Reply:
I really like training, because, the interfaces _do_ the training...it describes the page by what it is used for, not by a technical PL lingo term.

Reply:
I agree. 'Gradients and Training' will mean something to new users looking to learn PL, and as a side effect they'll learn what 'interface' means.

Whereas 'Gradients and Interfaces' is a bit more esoteric sounding.